src/components/ProjectSettingsTabs/GithubCommitQueue/getFormSchema.ts

```diff
@@ -0,0 +1,350 @@
+import { Field } from "@rjsf/core";
+import { SpruceFormProps } from "components/SpruceForm";
+import { CardFieldTemplate } from "components/SpruceForm/FieldTemplates";
+import widgets from "components/SpruceForm/Widgets";
+import { AliasRow } from "../AliasRow";
+import { overrideRadioBox, radioBoxOptions } from "../utils";
+import { FormState } from "./types";
+
+export const getFormSchema = (
+  useRepoSettings: boolean,
+  gitHubWebhooksEnabled: boolean,
+  formData: FormState,
+  repoData?: FormState
+): {
+  fields: Record<string, Field>;
+  schema: SpruceFormProps["schema"];
+  uiSchema: SpruceFormProps["uiSchema"];
+} => ({
+  fields: {},
+  schema: {
+    definitions: {
+      aliasArray: {
+        type: "array" as "array",
+        items: {
+          type: "object" as "object",
+          properties: {
+            id: {
+              type: "string" as "string",
+            },
+            alias: {
+              type: "string" as "string",
+            },
+            variant: {
+              type: "string" as "string",
+              default: "",
+            },
+            variantTags: {
+              type: "array" as "array",
+              title: "Variant Tags",
+              items: {
+                type: "string" as "string",
+                title: "Variant Tag",
+                default: "",
+              },
+            },
+            task: {
+              type: "string" as "string",
+              default: "",
+            },
+            taskTags: {
+              type: "array" as "array",
+              items: {
+                type: "string" as "string",
+                title: "Task Tag",
+                default: "",
+              },
+            },
+          },
+        },
+      },
+    },
+    type: "object" as "object",
+    properties: {
+      github: {
+        type: "object" as "object",
+        title: "GitHub",
+        properties: {
+          gitHubWebhooksEnabled: {
+            type: "null",
+            title: "GitHub Webhooks",
+            description: `GitHub webhooks ${
+              gitHubWebhooksEnabled ? "are" : "aren't"
+            } enabled.`,
+          },
+          prTestingEnabledTitle: {
+            type: "null",
+            title: "GitHub Pull Request Testing",
+            ...(useRepoSettings === undefined && {
+              description:
+                "If enabled, then untracked branches will also use the file patterns defined here for PR testing.",
+            }),
+          },
+          prTestingEnabled: {
+            type: ["boolean", "null"],
+            oneOf: radioBoxOptions(
+              ["Enabled", "Disabled"],
```

Comment:
Wanted to call out that this behaves slightly different than the below (disabling Pull Request Testing doesn't prevent you from modifying github aliases).
![image](https://user-images.githubusercontent.com/26798134/150868507-6ee6dffb-bff1-4e1a-b066-1eb0daab4dbc.png)